frontend/src/__tests__/cypress/cypress/e2e/home/homeAdmin.cy.ts

```diff
@@ -0,0 +1,118 @@
+import { initHomeIntercepts, setAdminUser } from '~/__tests__/cypress/cypress/e2e/home/homeUtils';
+import { mockDocs } from '~/__mocks__/mockDocs';
+import { mockComponents } from '~/__mocks__/mockComponents';
+import { mockQuickStarts } from '~/__mocks__/mockQuickStarts';
+import { customServingRuntimesIntercept } from '~/__tests__/cypress/cypress/e2e/customServingRuntimes/customServingRuntimesUtils';
+import { notebookImageSettings } from '~/__tests__/cypress/cypress/pages/notebookImageSettings';
+
+describe('Home page Admin section', () => {
+  beforeEach(() => {
+    cy.interceptOdh('GET /api/docs', mockDocs());
+    cy.interceptOdh('GET /api/components', null, mockComponents());
+    cy.interceptOdh('GET /api/quickstarts', mockQuickStarts());
+  });
+  it('should show the admin section for admins', () => {
+    initHomeIntercepts({ disableHome: false, disableProjects: true });
+    setAdminUser(true);
```

Comment:
Christian recently merged this change, I think that's what you're after here.
```suggestion
    asProductAdmin();
```

If I am right, can you change all `setAdminUser` function calls and remove the utility? (see [the user calls here](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/__tests__/cypress/cypress/utils/users.ts))